app/components/a_level_row_component.html.erb

```diff
@@ -1,7 +1,28 @@
 <% if course.a_levels_requirements_answered? %>
-  <p class="govuk-summary-list-value govuk-!-margin-top-0">
-    <%= a_level_requirement_content %>
-  </p>
+  <% if @course.a_level_requirements.present? %>
+    <% Array(@course.a_level_subject_requirements).map do |a_level_subject_requirement| %>
```

Comment:
I put the Array here as defensive as I think it always would be []. Let me know your thoughts.